setup.py

```diff
@@ -67,9 +67,10 @@
     url='http://gitless.com',
     packages=['gitless', 'gitless.cli'],
     install_requires=[
-      'pygit2>=0.24.0',
-      'clint>=0.3.6',
-      'sh>=1.11' if sys.platform != 'win32' else 'pbs>=0.11'
+      # make sure it matches requirements.txt 
```

Comment:
I think that in `install_requires` you want to specify the minimum versions required and not pin a specific version (e.g., `pygit2==0.26.4` should be `pytgit2>=0.26.4`)

https://packaging.python.org/discussions/install-requires-vs-requirements/

Reply:
There is a big warning on `pygit2` side that says:

> Backwards compatibility is not guaranteed even between micro releases. Please check the release notes for incompatible changes before upgrading to a new release.

https://www.pygit2.org/install.html#version-numbers

So I thought we need to be on a safe side for every release.

Reply:
Ok, perhaps we should only pin `pygit2` and leave the other ones as they were before

Reply:
As they were in `requirements.txt` or in `setup.py`?

Reply:
`setup.py`